main.go

```diff
@@ -24,7 +26,12 @@ func (c *CLI) Run() error {
 		return err
 	}
 
-	return function.Serve(&Function{log: log},
+	return function.Serve(
+		&Function{
+			log: log,
+			// Note: This does not work under windows (do we even support it?)
+			fsys: os.DirFS("/"),
```

Comment:
🤔 I don't think we do. Would using Afero solve the issue?

Reply:
I think it would because it just uses `os.Open` underneath (ref. https://github.com/spf13/afero/blob/v1.11.0/os.go#L55). Though this would mean including another dependency that would otherwise not be necessary and I would like to avoid that.

And then again having an `fs.FS` implementation that uses `os` and works on Windows is only so much code:

```go
var _ fs.FS = &OsFS{}

type OsFS struct {}

func (_ *OsFS) Open(name string) (fs.File, error) {
	return os.Open(name)
}
```

Reply:
On a second though: I think we can go with the small code above since it is nothing more than just a wrapper around a standard function. I will update the code.

This will then also work on Windows (if ever required).